src/plugins/index_pattern_management/public/components/index_pattern_table/empty_state/empty_state.tsx

```diff
@@ -0,0 +1,234 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import './empty_state.scss';
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { DocLinksStart, ApplicationStart } from 'kibana/public';
+import {
+  EuiPageContentHeader,
+  EuiPageContentHeaderSection,
+  EuiTitle,
+  EuiPageContentBody,
+  EuiPageContent,
+  EuiIcon,
+  EuiSpacer,
+  EuiFlexItem,
+  EuiDescriptionList,
+  EuiFlexGrid,
+  EuiCard,
+  EuiLink,
+  EuiText,
+} from '@elastic/eui';
+import { useHistory } from 'react-router-dom';
+import { reactRouterNavigate } from '../../../../../../plugins/kibana_react/public';
+import { MlCardState } from '../../../types';
+
+export const EmptyState = ({
+  onRefresh,
+  navigateToApp,
+  docLinks,
+  getMlCardState,
+  canSave,
+}: {
+  onRefresh: () => void;
+  navigateToApp: ApplicationStart['navigateToApp'];
+  docLinks: DocLinksStart;
+  getMlCardState: () => MlCardState;
+  canSave: boolean;
+}) => {
+  const mlCard = (
+    <EuiFlexItem>
+      <EuiCard
+        onClick={() => navigateToApp('ml', { path: '#/filedatavisualizer' })}
+        className="inpEmptyState__card"
+        betaBadgeLabel={
+          getMlCardState() === MlCardState.ENABLED
+            ? undefined
+            : i18n.translate(
+                'indexPatternManagement.createIndexPattern.emptyState.basicLicenseLabel',
+                {
+                  defaultMessage: 'Basic',
+                }
+              )
+        }
+        betaBadgeTooltipContent={i18n.translate(
+          'indexPatternManagement.createIndexPattern.emptyState.basicLicenseDescription',
+          {
+            defaultMessage: 'This feature requires a Basic license.',
```

Comment:
Should we say "free" Basic license?

Er actually, I thought ML was platinum?

Reply:
`x-pack/plugins/ml/common/license/ml_license.ts` - seems licensing in the ml plugin is more complex than that.

Reply:
The file data visualizer (file uploader) and the index data visualizer are available on basic.
Anomaly detection etc is platinum only.